### PR TITLE
Update Inkeep quick questions and search tabs

### PR DESCRIPTION
**Pull request by kavalerov**
## Description

This PR updates quick questions in Inkeep and adds Docs and Blog tabs to the search results

### CODE REVIEW

File: src/external-scripts/inkeep.ts [hunk 1]
```diff
@@ -47,9 +47,9 @@ const aiChatSettings = {
       },
     ],
     quickQuestions: [
-      'How do I get presence data for a specific member?',
-      "What's the difference between detach and unsubscribe?",
-      "Can I limit users' access to message interactions?",
+      'What is a channel?',
+      "How do I authenticate with Ably?",
+      "How to manage user status?",
```
Comment (m-hulbert) [inline src/external-scripts/inkeep.ts:52]: Small thing: but I'd keep this consistent with the authentication question and make it 'How do I manage user status?'

Comment (kavalerov) [reply src/external-scripts/inkeep.ts]: Fixed in c16bb8ca71e382531572b4724171be429d7e26a5

**Comment by kavalerov** [comment]
@matt423 @m-hulbert I have added copy chat button and updated Inkeep version to support it - hope it is ok to bundle in one PR

File: src/external-scripts/inkeep.ts (hunk 1 above)
Comment (m-hulbert) [reply src/external-scripts/inkeep.ts]: Got a consistent linting error. Shout if you want me to take a look 🙂

Comment (kavalerov) [reply src/external-scripts/inkeep.ts]: No it is my bad :( sorry, was rushing yesterday with this.

**Pull request merged**